.gitignore

```diff
@@ -17,6 +17,7 @@ dist
 .DS_Store
 .env
 .eslintrc
+.package-lock.json
```

Comment:
Lock file should not be ignored

Reply:
Had to change from Linux to Windows so I figured better safe than sorry xD Corrected